tools/IoTEdgeDevOps/IoTEdgeDashboard/Startup.cs

```diff
@@ -57,7 +57,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 {
                     routes.MapRoute(
                         name: "default",
-                        template: "{controller=Agent}/{action=Index}/{id?}");
+                        template: "{controller=Home}/{action=Index}");
```

Comment:
Should we consider using AspNet session feature? 
https://www.c-sharpcorner.com/UploadFile/484ad3/session-state-in-Asp-Net/

Reply:
Please specify what do you want to use AspNet session for?

Reply:
Per the document, I can foresee using a session as a cache as we enable more components to the dashboard. The session is not relevant for our application at the moment.

Reply:
Ok, for user session, I think this may not be something applicable to use at least for now.  For caching, we can think about it later on and handle in another PBI.